src/components/Dialog-Window/tabs/tabs.stories.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from "react";
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+import { Tabs } from "./tabs";
+
+export default {
+  title: "DialogContent/DialogTabs",
```

Comment:
I think we should try to avoid using main categories that aren't Atoms, Molecules, and so on. If it's vital, perhaps we could use `Dialog content` as a sub category instead?

```suggestion
  title: "Organisms/Dialog content/Dialog tabs",
```